.github/workflows/build-wheels-windows.yml

```diff
@@ -0,0 +1,49 @@
+name: windows-dispatch
+
+# TODO schedule for workflow and delete pull_request/push
```

Comment:
Not necessarily but it would be better to delete every automatic run until it is completed. So yes, I will delete it and leave only `on: workflow_dispatch` before merge for now.